macos_postinstall/src/main.rs

```diff
@@ -0,0 +1,129 @@
+use ini::Ini;
+use itertools::sorted;
+use lazy_static::lazy_static;
+use std::{
+    env,
+    fs::{self, File},
+    io::Write,
+    sync::Mutex,
+};
+
+const LOG_FILE: &str = "/tmp/postinstall_MongoDB_Atlas_SQL_ODBC.log";
```

Comment:
I found that my tmp directory on mac isn't just /tmp/ but something more cumbersome (kind of like Windows).
@terakilobyte Do you know if it matters? Because if we can write directly into /tmp/ it seems just as good.
Otherwise, maybe we want to use what we have in the ODBC logger
`env::temp_dir();`